e2e/specs/modals.spec.js

```diff
@@ -97,6 +97,19 @@ describe("modals", () => {
     );
   });
 
+  it("renders the video recorded dialog width correctly", () => {
+    // Set the viewport to 1280 x 720 so checking for 80vw cleaner
```

Comment:
What does "checking for 80vw cleaner" mean?

Reply:
My bad that comment doesnt make much sense lol - will update.
Context: (80% * `1366px` the standard viewport width we use in Cypress tests = `1092.8px` which is not a easy round number for testing).